source/includes/steps-enable-authentication-in-shardcluster-nodowntime-auth-mongos.yaml

```diff
@@ -0,0 +1,53 @@
+stepnum: 1
+level: 4
+title: "Remove ``transitionToAuth`` from the ``mongos`` configuration file"
+ref: modify-config-files
+pre: |
+
+  Remove the :setting:`security.transitionToAuth` key and its value from the
+  configuration file created during this tutorial.
+  
+  .. code-block:: yaml
+
+     security:
+        keyFile: <path-to-keyfile>
+
+---
+stepnum: 2
+level: 4
+ref: restart-mongos
+title: "Restart the ``mongos`` with the updated configuration file."
+pre: |
+
+  Connect to the :program:`mongos` and use the :method:`db.shutdownServer()`
+  method against the ``admin`` database to safely shut down the
+  :program:`mongos`.
+  
+  Restart one :program:`mongos` at a time to ensure that clients
+  can connect to the sharded cluster. If your cluster has only one
+  :program:`mongos`, this step results in downtime while the :program:`mongos`
+  is offline.
```

Comment:
The ordering of the sentences in this step are slightly wonky.

.. note:: If your cluster has only one mongos ...

Connect to the mongos ...

.. code-block:: 

...
Restart (not you can then restart since you're not giving people a choice -- they need to restart) the mongos specifying ...